common/djangoapps/microsite_configuration/backends/filebased.py

```diff
@@ -0,0 +1,199 @@
+"""
```

Comment:
Can we rename this module to something more like file_system.py, to match with the more noun-oriented database.py?